README.md

```diff
@@ -1,2 +1,49 @@
 # openshift-servicemesh-plugin
 Webpack Plugin to integrate Kiali into OpenShift Console
+
+## Platform Setup
+
+1. OpenShift +4.10 cluster with OpenShift ServiceMesh / Istio installed. 
+2. Kiali deployed in the cluster
+3. "oc" client available in the path
+4. "podman" client available in the path
+
+## Deploy the Service Mesh Plugin
+
+```sh
+# Login in OpenShift cluster i.e oc login ...
+
+# Adjust the kialiUrl in the "plugin-conf" ConfigMap under plugin/manifest.yaml pointing to the Kiali public URL
+# i.e. https://kiali-istio-system.apps-crc.testing 
+
+make deploy-plugin
+make enabled-plugin 
```

Comment:
```suggestion
make enable-plugin 
```

Reply:
This is what I see - notice the "ServiceMesh" tab in the OS Console:
![image](https://user-images.githubusercontent.com/2029470/170565582-597da616-54a7-43e1-8f65-bbcb907abd29.png)